src/app/homepage/homepage.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="container">
+  <div class="header">
+    <button class="register-btn">Registration</button>
+    <button class="login-btn">Log In</button>
+  </div>
+  <div class="banner">
+  </div>
+  <div class="three-phrases">
+    <div class="organizer">travel organizer</div>
+    <div class="easy">easy to use</div>
+    <div class="forget">forget nothing</div>
+  </div>
+  <div class="todo">
+    <i class="material-icons">add</i>
+    <span class="add-todo">
+      create your first travel ToDo
+    </span>
+  </div>
+  <app-footer></app-footer>
```

Comment:
Instead of adding the footer to the homepage you could check if you want to do one of the options here.
https://blog.angularindepth.com/angular-routing-reusing-common-layout-for-pages-from-different-modules-440a23f86b57

Then you have to declare the footer only once.

Reply:
Thanks @MyracleDesign. I took it out and put it in the app component.